examples/transforms/scale.rs

```diff
@@ -41,7 +41,7 @@ fn setup(
         .spawn_bundle(PbrBundle {
             mesh: meshes.add(Mesh::from(shape::Cube { size: 1.0 })),
             material: materials.add(Color::WHITE.into()),
-            transform: Transform::from_translation(Vec3::ZERO),
+            transform: Transform::from_rotation(Quat::from_rotation_y(PI / 4.0)),
```

Comment:
You didn't import PI

Reply:
If I'm correct bevy won't need the explicit .system() call in future? Should I remove those too? (with the merged main)

Reply:
It seems they removed the `.system()` in the other examples, so yes indeed, nice catch :)